cmd/auth/auth.go

```diff
@@ -34,25 +37,29 @@ GCP: https://docs.gcp.databricks.com/en/dev-tools/auth/index.html`,
 }
 
 func promptForHost(ctx context.Context) (string, error) {
+	if !cmdio.IsInTTY(ctx) {
+		return "", fmt.Errorf("the command is being run in a non-interactive environment, please specify a host using --host")
+	}
+
 	prompt := cmdio.Prompt(ctx)
-	prompt.Label = "Databricks Host (e.g. https://<databricks-instance>.cloud.databricks.com)"
-	// Validate?
-	host, err := prompt.Run()
-	if err != nil {
-		return "", err
+	prompt.Label = "Databricks Host"
+	prompt.Validate = func(host string) error {
+		if !strings.HasPrefix(host, "https://") {
+			return fmt.Errorf("host URL must have a https:// prefix")
+		}
+		return nil
 	}
-	return host, nil
+	return prompt.Run()
 }
 
 func promptForAccountID(ctx context.Context) (string, error) {
+	if !cmdio.IsInTTY(ctx) {
+		return "", errors.New("the command is being run in a non-interactive environment, please specify an account ID using --account-id")
+	}
+
 	prompt := cmdio.Prompt(ctx)
 	prompt.Label = "Databricks Account ID"
 	prompt.Default = ""
 	prompt.AllowEdit = true
-	// Validate?
```

Comment:
This is always a UUID AFAIK, so it could be validated (optional).

Reply:
added validation.